### ISSUE TITLE
int ndarray becomes float when converted to Tensor

### ISSUE THREAD

**Issue by iArunava** [pytorch/pytorch#18679]
## 🐛 Bug

int ndarray becomes normalized floats when converted to Tensor

## To Reproduce


```
>>> import torch
>>> import numpy as np
>>> from PIL import Image
>>> import torchvision.transforms as trfs

>>> i = Image.open('./vault/camvid/labels/_labels/0001TP_006690_P.png')
>>> np.unique(i)
array([ 4,  5,  8, 10, 12, 14, 16, 17, 19, 21, 22, 24, 26, 27, 30], dtype=uint8)

>>> t = trfs.Compose([trfs.ToTensor()])
>>> ii = t(i)
>>> torch.unique(ii)
tensor([0.0392, 0.0667, 0.0745, 0.0549, 0.0627, 0.0863, 0.0471, 0.0196, 0.1020, 0.1059, 0.0941, 0.1176, 0.0824, 0.0314, 0.0157])
```
~                                                                                                                                                                                                                                              
~                                               

## Expected behavior
```
>>> torch.unique(ii)
tensor([ 4,  5,  8, 10, 12, 14, 16, 17, 19, 21, 22, 24, 26, 27, 30])
```


## Environment

 - PyTorch Version (e.g., 1.0): 1.0.1
 - OS (e.g., Linux): Elementary OS
 - How you installed PyTorch (`conda`, `pip`, source): pip
 - Build command you used (if compiling from source):
 - Python version: 3.6
 - CUDA/cuDNN version: NA
 - GPU models and configuration: NA
 - Any other relevant information: NA

## Additional context

Is this an intended behaviour? Or I am missing something?

**Comment by fmassa** [comment]
This is not due to pytorch, but to torchvision. And the behavior is expected, because we convert uint8 images to float sensors in 0-1

**Comment by iArunava** [comment]
Okay thanks for the info :)
